tests/AntDesign.Tests/Select/Actions/Select.ClearTests.razor

```diff
@@ -0,0 +1,118 @@
+﻿@using System.Net.Sockets
+@inherits AntDesignTestBase
+@code {
+    record Person(int Id, string Name);
+    List<Person> _persons = new()
+    {
+        new(1, "John"),
+        new(2, "Lucy"),
+        new(3, "Jack"),
+        new(4, "Emily"),
+    };
+
+    [Fact]
+    public void DefaultSelectWithoutAllowClear()
+    {
+        JSInterop.Setup<DomRect>(JSInteropConstants.GetBoundingClientRect, _ => true)
+            .SetResult(new DomRect());
+        var cut = Render<Select<int, Person>>(
+            @<AntDesign.Select DataSource="@_persons"
+                               LabelName="@nameof(Person.Name)"
+                               ValueName="@nameof(Person.Id)"
+                               Value="0">
+            </AntDesign.Select>);
+        Assert.Throws(typeof(ElementNotFoundException), () => cut.Find("span.ant-select-clear"));
+    }
+
+    [Fact]
+    public void DefaultSelectWithAllowClear()
+    {
+        JSInterop.Setup<DomRect>(JSInteropConstants.GetBoundingClientRect, _ => true)
+            .SetResult(new DomRect());
+        var value = 1;
```

Comment:
There are plans in the future to redo the way clear button works (or rather how it changes the value). Safest approach would be to change it to 
```csharp
int? value = 1;
```
and evaluate later 
```csharp
value.Should().Be(default(int?));
```

Reply:
This will cause the unit test to fail. I think we can temporarily keep it and modify it in the future, or just delete this test

Reply:
Here's the full adjustment for the test to pass:
```razor
    [Fact]
    public void DefaultSelectWithAllowClear()
    {
        JSInterop.Setup<DomRect>(JSInteropConstants.GetBoundingClientRect, _ => true)
            .SetResult(new DomRect());
        int? value = 1;
        var cut = Render<Select<int?, Person>>(
            @<AntDesign.Select
                 DataSource="@_persons"
                 ValueName="@nameof(Person.Id)"
                 LabelName="@nameof(Person.Name)"
                 @bind-Value="@value"
                 AllowClear/>
            );
        var clear = cut.Find("span.ant-select-clear");
        clear.Click();
        value.Should().Be(default(int?));
    }
```